### ISSUE TITLE
UI doesn't pull callbacks

### ISSUE THREAD

**Issue by who1smrrobot** [its-a-feature/Mythic#149]
Dear Mythic-Developer-Team,

first of all. amazing work of you guys!

I just updated my Mythic instance to the latest version and currently run sometimes into this strange issue:

Callback -> task an agent (e.g. `whoami`) is being submitted to /graphql API. However the task does not appear on the task list (see image below). When creating new tasks they immediately appear on the task list and get updated.

![image](https://user-images.githubusercontent.com/41398783/152319052-83cb6b37-28fa-405f-9713-1c929451f887.png)

Only after refreshing the browser with F5 the missing item (in my case the whoami from February 3rd, 2022 09:39:12) will be displayed (see image below)

![image](https://user-images.githubusercontent.com/41398783/152319084-e13b5eb7-6eee-4609-95fb-3c7d3f799c07.png)

**Comment by its-a-feature** [comment]
Thanks for the report! I don't think I've seen that happen before - is this something you're able to keep reproducing? If I understand correctly, this is the process to reproduce:

- click the callback to open the tab at the bottom
- issue a task (it doesn't show up)
- issue another task (it does show up)
- refresh the page (now they both show up)

is that correct?

**Comment by who1smrrobot** [comment]
That is correct. 

One additional info: I just tried it with the Medusa agent and same issue there... so maybe this issue should be moved to the Mythic repo.

![image](https://user-images.githubusercontent.com/41398783/152674319-ccfb0f8c-906e-4905-a5d7-e8b0556de5c8.png)

![image](https://user-images.githubusercontent.com/41398783/152674336-93ebcfc7-0530-4427-a5d1-1b3dfe81bf42.png)

**Comment by its-a-feature** [comment]
does this happen every time you click interact? It doesn't seem to be consistent either (one screenshot shows you missing the first task, another shows a middle task). Is it always just the first task after you happen to click interact that it fails to load? I haven't been able to replicate this, what browser are you using?

**Comment by who1smrrobot** [comment]
Unfortunately not. It happens super random and I tried in the last days to get a reproducable PoC of this issue, but no success so far.

I have tried so far the following browsers:
* Windows 11 - Edge 97.0.1072.76 
* Windows 10 - Vivaldi (Chromium 5.0.2497.48)
* Kali/Debian - Firefox 78.14.0esr

**Comment by who1smrrobot** [comment]
I have the feeling that this issue occurs, once the WebSocket connection crashed/failed for whatever reason:

![image](https://user-images.githubusercontent.com/41398783/152995932-ca3297ca-9488-4cd3-bf7f-40ca22de340c.png)

![image](https://user-images.githubusercontent.com/41398783/152997175-4294d4c7-cfd3-47d3-88e0-891751e1e1ce.png)


This could explain why it is hard to replicate it...

**Comment by its-a-feature** [comment]
I've adjusted how these tasks are streamed/fetched from the Mythic server in the UI that should help with this. If you're still experiencing this in UI version  v0.1.0rc25 or above, please re-open this issue and we can dive into it again
